infrastructure/templates/public-api/application/etlFunctionApp.bicep

```diff
@@ -0,0 +1,140 @@
+@description('Specifies the Subscription to be used.')
+param subscription string
+
+@description('Specifies the location for all resources.')
+param location string
+
+//Specific parameters for the resources
+@description('Application : Insights name')
+param applicationInsightsName string
+
+@description('Function App Plan operating system')
+@allowed([
+  'Windows'
+  'Linux'
+])
+param appServiceplanOS string = 'Linux'
+
+@description('Function App : Runtime Language')
+@allowed([
+  'dotnet'
```

Comment:
I believe we just need `dotnet`. I think it's a DFE requirement? @ntsim can confirm

Reply:
Agreed, but options left in for training purposes